tips/TIP-0048/openapi3-core.yaml

```diff
@@ -0,0 +1,3807 @@
+              slotsPerEpochExponent: 13
+              stakingUnbondingPeriod: 10
+              validationBlocksPerSlot: 10
+              punishmentEpochs: 10
+              livenessThresholdLowerBound: 15
+              livenessThresholdUpperBound: 30
+              minCommittableAge: 10
+              maxCommittableAge: 20
+              epochNearingThreshold: 24
+              congestionControlParameters:
+                minReferenceManaCost: "1"
+                increase: "0"
+                decrease: "0"
+                increaseThreshold: 800000
+                decreaseThreshold: 500000
+                schedulerRate: 100000
+                maxBufferSize: 1000
+                maxValidationBufferSize: 100
+              versionSignalingParameters:
+                windowSize: 7
+                windowTargetRatio: 5
+                activationOffset: 7
+              rewardsParameters:
+                profitMarginExponent: 8
+                bootstrappingDuration: 1154
+                manaShareCoefficient: "2"
+                decayBalancingConstantExponent: 8
+                decayBalancingConstant: "1"
+                poolCoefficientExponent: 11
+                retentionPeriod: 684
+              targetCommitteeSize: 32
+              chainSwitchingThreshold: 2
+        baseToken:
+          name: Shimmer
+          tickerSymbol: SMR
+          unit: SMR
+          subunit: glow
+          decimals: 6
+        features:
+          - test
+
+    get-buildingBlock-response-example:
+      value:
+        strongParents:
+          - "0x0482f0eba39a23c9a13072c93d828b55543132c47f5f57514d9e55535e9d4f4f35000000"
+          - "0xae7c4f55a6db8bf4841e4a38f06d32ab9bd88b927a6ba0bc19bcb19c625ff8b63c000000"
+        weakParents:
+          - "0x67e57f7446b5a6f152afabb17c6077c26512278a275310a7ff2fa513f4e0b7383d000000"
+          - "0x2e65c319e9a2c4a6ff7195f9a1ed896c43d0dded1b906979316d502b158965c23d000000"
+        shallowLikeParents:
+          - "0xe5fe5231630afaaba609af76787ff1ec9c6088dd17e9cf922152b3facd7bd5883e000000"
+          - "0xed4b771b5413f5118dd80021ca07fb727e4c54eec9d28d6566c28cc81e7d267c3f000000"
+        latestParentBlockIssuingTime: "1690879505000000000"
+        latestFinalizedSlot: 1422
+        latestCommitment:
+          protocolVersion: 3
+          slot: 1432
+          previousCommitmentId: "0xc0300d55fe2cdc93191e52685da009f53d40883ae111291e550eed4e7eb7752746010000"
+          rootsId: "0x6a8f424929e1f08d87a204efc8a60499b789465aaa28178b77debe75cc2915d3"
+          cumulativeWeight: '45678'
+          referenceManaCost: '12345'
+
+    post-blocks-response-example:
+      value:
+        blockId:        "0x0482f0eba39a23c9a13072c93d828b55543132c47f5f57514d9e55535e9d4f4f35000000"
+
+    get-block-by-id-response-example-new-transaction:
+      value:
+        blockId: "0x0482f0eba39a23c9a13072c93d828b55543132c47f5f57514d9e55535e9d4f4f35000000"
+        blockState: "pending"
+        transactionState: "pending"
```

Comment:
This is not up to date, should be contained in a `transactionMetadata`.
Please check other occurrences of that 🙏🏻